lib/new-admin/middlewares/session.js

```diff
@@ -19,12 +20,16 @@ const getSecret = () => {
 
 const hostname = options.hostname
 
+const lamaDb = {
+  query: (query, values, qrm) => db.query(query, values, qrm, false)
+}
```

Comment:
What's this doing?

Reply:
I believe it's older code from this PR that I haven't reverted